examples/transparent_layer_bar.json

```diff
@@ -0,0 +1,20 @@
+{
+  "data": { "url": "data/population.json",
```

Comment:
Maybe add description:

``"description": "..."`

(Think if you're putting this on a newspaper or something, what would the caption of the output figure.)